lib/Driver/DarwinToolChains.cpp

```diff
@@ -712,6 +712,11 @@ toolchains::Darwin::constructInvocation(const DynamicLinkJobAction &job,
   Arguments.push_back("-arch");
   Arguments.push_back(context.Args.MakeArgString(getTriple().getArchName()));
 
+  if (context.cxxInteropEnabled()) {
+    // We only support libc++ on Darwin.
```

Comment:
This is an ambiguous comment. It could mean either of these:

1) We don't support other standard library implementations on Darwin.
2) We don't support other platforms.

I think you mean the former?

Reply:
Yes -- thanks for pointing out the ambiguity! I've changed this to "On Darwin, we only support libc++." WDYT?

Reply:
SG